.github/workflows/lint_test_charts.yaml

```diff
@@ -86,8 +86,6 @@ jobs:
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
           start args: "--container-runtime=containerd"
-      - name: Create test ns
-        run: kubectl create ns ct
```

Comment:
The thing on testing in a namespace (`ct` in this pipeline) it to test that we are installing it a namespace that is neither `default` nor `newrelic` so you can test that all the objects are created in a different namespace and are able to boot.

I don't get why we need to remove the namespace. I don't get the race condition we are fixing... Why without it we remove the race condition?

Reply:
Without `--namespace ns` selects a random namespace that it is then deleted:
```
namespace "super-agent-6s2970ilqu" deleted
```

By doing so we are:
 - covering both that we are installing in a namespace different from default nor newrelic
 - having "for free" a `kubectl delete --wait` it waits for all resources to be deleted (sadly we cannot add it as a flag of `Helm delete`)

Reply:
> selects a random namespace that it is then deleted

TIL. Sounds awesome then :D

Reply:
Cool, I learned something new too